app/src/main/java/ch/hikemate/app/ui/navigation/NavigationActions.kt

```diff
@@ -0,0 +1,100 @@
+package ch.hikemate.app.ui.navigation
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.Menu
+import androidx.compose.material.icons.outlined.Person
+import androidx.compose.material.icons.outlined.Place
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.navigation.NavGraph.Companion.findStartDestination
+import androidx.navigation.NavHostController
+
+/** Object containing route constants. */
+object Route {
+  const val OVERVIEW = "Overview"
+  const val MAP = "Map"
+  const val AUTH = "Auth"
+  const val PROFILE = "Profile"
+}
```

Comment:
Nitpicking: Here, I think the Overview route would be the same as the Map route. As you can see in the Figma, it's on the same screen and so on the same route. Tell me what you think.

Reply:
I actually disagree on this one. I don't understand why the overview and the map would be two different routes, since clearly the overview and the map are two different screens even on the figma.
Maybe there is a confusion in either side. 

My understanding behind it is:
- The `Overview` shows the list of hikes that have already been marked as booked.
- The `Map` shows the possible hikes the user can take.

Let me know if I am getting something wrong.

Reply:
PD: Same goes for all the related comments on the topic.

Reply:
After looking at the Figma maybe (let me know if I am getting it wrong) you are confusing the `Overview` with the `HikePreview` of the Mockup. For the navigation I based it on the wireframe.

Reply:
I think it's a big misunderstanding.
My understanding is:
- The `Overview` is the page with the map on the top and the list on the bottom
- The `Map` would then not need a route (maybe later if we want a screen with just the map)

Maybe we should ask the other about what they think.